aptos-move/framework/aptos-framework/sources/primary_store.move

```diff
@@ -0,0 +1,156 @@
+/// This defines the module for interacting with primary stores of accounts/objects, which have deterministic addresses
+module aptos_framework::primary_store {
```

Comment:
Why is store > wallet?

Reply:
from different ppl including our metapixel friends. I think store is better than container since they are used to xxx_store already.